tests/unit/test_cost.py

```diff
@@ -161,7 +165,7 @@ def test_pybamm_design_costs(self, pybamm_costs, model, dataset, one_parameter):
         problem.set_params(np.array([0.52]))
         lower_cost = problem.run()
 
-        assert higher_cost > lower_cost
+        assert higher_cost < lower_cost  # Optimising negative cost
```

Comment:
very small thing, but can we just rename `higher_cost` -> `lower_cost` and visa-versa?